pkg/provider/azure_loadbalancer.go

```diff
@@ -3341,6 +3359,41 @@ func unbindServiceFromPIP(pip *network.PublicIPAddress, service *v1.Service, ser
 	return nil
 }
 
+// getBackendPrivateIPs returns the private IPs of the LoadBalancer's backend pool
```

Comment:
Can we put this in azure_backendpools.go and implement for both IPConfig and IP based backend pool?

Reply:
Thanks for that! Refactored, please review